README.md

```diff
@@ -592,6 +592,7 @@ Table of Contents
   * [onelogin.com](https://www.onelogin.com/) — Identity as a Service (IDaaS), Single Sign-On Identity Provider, Cloud SSO IdP, 3 company apps and 5 personal apps, unlimited users
   * [Operous](https://operous.dev/) — Cloud instance testing tool with a comprehensive and automated set of test-suites of best practices, performance, and security. Free tier offers 100 testing minutes, 10 Test Suites, and up to 5 instances to 1 user.
   * [opswat.com](https://www.opswat.com/) — Security Monitoring of computers, devices, applications, configurations,... Free 25 users and 30 days history users.
+  * [Oso](https://cloud-docs.osohq.com/) - Application Authorization. Authorization-as-a-service providing abstractions for building and iterating on authorization in your application. Free access to Oso Cloud with a Developer Account.
```

Comment:
Do we know if they will be free also after beta?

Reply:
yes, there will be a free Developer tier after beta.

Reply:
Please show a link detailing that

Reply:
seems like its no more in beta...